README.md

```diff
@@ -46,17 +46,17 @@ You have to open (or create a new) project folder where you have to define a `la
 
 ```json
 {
-  "version": "0.2.0",
-  "configurations": [
-    {
-      "name": "Escargot: Attach",
-      "type": "escargot",
-      "request": "attach",
-      "address": "localhost",
-      "localRoot": "${workspaceRoot}",
-      "debugLog": 0
-    },
-  ]
+    "configurations": [
+       {
+         "name": "Escargot: Attach",
+         "type": "escargot",
+         "request": "attach",
+         "address": "localhost",
+         "debugServer": 4711,
```

Comment:
This is only for development purposes, so it is unnecessary in the default config .